src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -1214,18 +1214,20 @@ public static final Correspondent getEudora() {
    * @param ascensionClass The name of the character's class
    */
   public static final void setAscensionClass(final AscensionClass ascensionClass) {
-    KoLCharacter.ascensionClass = ascensionClass;
+    if (KoLCharacter.ascensionClass != ascensionClass) {
```

Comment:
Can I suggest you simply do this at the top of the method:
```
   if (KoLCharacter.ascensionClass == ascensionClass) {
      return;
   }
```
That way, you do not have to re-indent the rest of the method because it is inside a conditional.

Reply:
done :)